src/libs/ajax/leonardo/providers/LeoAppProvider.ts

```diff
@@ -0,0 +1,20 @@
+import { Ajax } from 'src/libs/ajax';
+import { ListAppResponse } from 'src/libs/ajax/leonardo/models/app-models';
+
+export interface LeoAppProvider {
+  listWithoutProject: (args: Record<string, string>, signal?: AbortSignal) => Promise<ListAppResponse[]>;
+  delete: (resourceName: string, appName: string, deleteDisk?: boolean, signal?: AbortSignal) => Promise<void>;
+  pause: (resourceName: string, appName: string, signal?: AbortSignal) => Promise<void>;
+}
+
+export const leoAppProvider: LeoAppProvider = {
```

Comment:
I realized I didn't include unit test coverage for these, which is needed since Environments page that uses them is mocking these more convenient interfaces in it's unit tests, so no free test coverage from there.  I will add unit tests to these providers today (Sept 7th)

Reply:
coverage added